server/templates/pages/src/privacy.html

```diff
@@ -1,8 +1,5 @@
 <!doctype html>
-<!--[if lt IE 7]>      <html class="no-js lt-ie10 lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
-<!--[if IE 7]>         <html class="no-js lt-ie10 lt-ie9 lt-ie8"> <![endif]-->
-<!--[if IE 8]>         <html class="lt-ie10 lt-ie9"> <![endif]-->
-<!--[if IE 9]>         <html class="lt-ie10"> <![endif]-->
+<!--[if lt IE 10]>     <html class="lt-ie10"> <![endif]-->
 <!--[if gt IE 9]><!--> <html dir="{{ lang_dir }}" lang="{{ lang }}"> <!--<![endif]-->
```

Comment:
Can you change `if gt IE 9` to `if gte IE 10`. Also, add the dir/lang atrributes to R2.